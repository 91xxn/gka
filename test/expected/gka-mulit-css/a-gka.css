.gka-wrap {
    width: 400px;
    height: 300px;
}

.animation {
    width: 400px;
    height: 300px;
    background-image: url("./img/a01.png");
    background-size: 400px 300px;
    background-repeat: no-repeat;
    animation-name: keyframes-a;
    animation-duration: 0.36s;
    animation-delay: 0s;
    animation-iteration-count: infinite;
    animation-fill-mode: forwards;
    animation-timing-function: steps(1);
}

@keyframes keyframes-a {
    0% {
        width: 400px;
        height: 300px;
        background-image: url("./img/a01.png");
        background-size: 400px 300px;
    }
    11.11% {
        background-image: url("./img/a02.png");
    }
    22.22% {
        background-image: url("./img/a03.png");
    }
    33.33% {
        background-image: url("./img/a04.png");
    }
    44.44% {
        background-image: url("./img/a05.png");
    }
    55.56% {
        background-image: url("./img/a06.png");
    }
    66.67% {
        background-image: url("./img/a07.png");
    }
    77.78% {
        background-image: url("./img/a08.png");
    }
    88.89%,
    100% {
        background-image: url("./img/a09.png");
    }
}