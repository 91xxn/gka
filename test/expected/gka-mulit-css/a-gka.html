<html>
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=1,user-scalable=0,maximum-scale=1">
    
    <title>gka-preview</title>
<link href="./a-gka.css" rel="stylesheet" type="text/css">
</head>
<body>

    <div class="gka-wrap">
        <div id="gka"></div>
    </div>
    <script>
    function preloadImage(names, cb, prefix){
        window.gkaCache = window.gkaCache || [];
        var n = 0,img,imgs = {};
        names.forEach(function(name) {
            img = new Image();
            window.gkaCache.push(img);
            img.onload = (function(name, img) {
                return function() {
                    imgs[name] = img;
                    (++n === names.length) && cb && cb(imgs);
                }
            })(name, img);
            img.src = (prefix || '') + name;
        });
    }

    preloadImage(["a01.png","a02.png","a03.png","a04.png","a05.png","a06.png","a07.png","a08.png","a09.png"], function() {
        document.getElementById('gka').className += " animation"
    }, "img/")
    </script>
</body>
</html>